bindings/pyfiction/include/pyfiction/documentation.hpp

```diff
@@ -3,4 +3,28 @@
 
 #include "pyfiction/pybind11_mkdoc_docstrings.hpp"
 
+static const char *__doc_fiction_cartesian_layout_overridden =
```

Comment:
Could you do the same for the hexagonal layouts? We use the `even_row` orientation in `pyfiction`. Many thanks!